releasenotes.txt

```diff
@@ -0,0 +1,314 @@
+BISDN Linux
+4.8.0 Release Notes
+Date: 2022-09-05
+
+! New in this release
+- added Edgecore EPS201 (AS4630-54TE) as supported platform
+- updated to Linux kernel 5.15.64
+- bundle MSTPD state as well in collect-debug-info
+
+! Changed behavior
+- install 90-enp.link to /lib/systemd/network instead of /etc/systemd/network to
+  avoid accidental deletions or changes by users
+- set default hostname based on ONIE machine name
+- the installer now checks if the platform is supported before installing
+- there is no default bond0 device created anymore
+
+! Bug fixes
+- avoid link flapping when DHCP sets MTU (only affects platform with Intel
+  management interfaces)
```

Comment:
Since all x86 platforms are intel based, might be more straight forward to just say

```suggestion
- avoid link flapping when DHCP sets MTU on x86 based platforms
```

and avoid people wondering whether their x86 based system has an Intel management interface or not.

Reply:
Instead of people wondering if their system has an intel card, they might wonder how x86 system arch is related to flapping links and DHCP.

Reply:
At least they can tell whether they are affected or not. Maybe "on platforms with Intel x86 host CPUs" is better?

Reply:
what is the releation between the host CPU architecture and link flapping?

Reply:
It's related to being an Intel CPU with an integrated MAC (well several)

Reply:
Host CPU is what this is usually called in the switch descriptions by the Vendors.

Reply:
done